Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
lol, i'm sorry this is such a disaster. FYI, poet ships a second tool named `poet_lint` that will accept a bunch of concatenated `resource` lists and sort them for you: https://github.com/tdsmith/homebrew-pypi-poet/blob/master/poet/lint.py